bitrixSM/internal/transport/handlers/webhook.go

```diff
@@ -0,0 +1,57 @@
+package handler
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+type Client struct {
+	WebhookURL string
+	HTTPClient *http.Client
+}
+
+func NewClient(webhookURL string) *Client {
+	return &Client{
+		WebhookURL: webhookURL,
+		HTTPClient: &http.Client{Timeout: 10 * time.Second},
+	}
+}
+
+func (c *Client) CallMethod(method string, params interface{}, result interface{}) error {
```

Comment:
it is your internal API - you can provide semantic methods for just your needs instead of such generic function.  I see, looks like it is similar to bitrix API, but it will leads to high coupling between external system (bitrix) and your internal system.